cli/organization.go

```diff
@@ -0,0 +1,86 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/coder/coder/v2/cli/clibase"
+	"github.com/coder/coder/v2/cli/cliui"
+	"github.com/coder/coder/v2/codersdk"
+)
+
+func (r *RootCmd) organizations() *clibase.Cmd {
+	cmd := &clibase.Cmd{
+		Annotations: workspaceCommand,
+		Use:         "organizations { current }",
+		Short:       "Organization related commands",
+		Aliases:     []string{"organization", "org", "orgs"},
```

Comment:
Should we instead do something like `coder ctx current`?

Reply:
I'd be interested in discussing this further, ultimately I'd like for us to support config files for the CLI too, you might not want same behavior across all orgs. Having this support those use-cases as well would be nice.

Reply:
Good point. This command is marked as hidden, so we can tweak it before we release.

Multi orgs is still a bit out, but this is going to be really helpful in development.